src/pages/home/containers/hero/Hero.jsx

```diff
@@ -1,8 +1,42 @@
-import { Wrapper } from "@components";
-import Button from "../../../../components/button/Button";
+import { Wrapper, Button} from "@components";
+import { headerData } from "./utils";
+import { IPlay } from "@assets";
+import "./hero.css";
+
 
 const Hero = () => {
-  return <div>Hero</div>;
+  const handleLearnMore = () => {};
+  const handleWatchteaser = () => {};
+  return (
+    <>
+    {/* hero header */}
+    <header className={`${headerData[0].classes}`}>
+      <p className="header-label text-white">{headerData[0].label}</p>
+      <div className="hero-wrap-title">
+        <h1 className="hero-title">{headerData[0].title}</h1>
+        <div className="hero-title-year">20<br/>24</div>
+      </div>
+      <h2 className="hero-title m-0">{headerData[0].subtitle}</h2>
+      <div className="sm:w-9/12 mt-4">
+        <p className="text-desc sm:tracking-widest ">{headerData[0].desc}</p>
+        <br />
+        <p className="text-desc sm:tracking-widest">{headerData[0].desc2}</p>
+      </div>
+    </header>
+    {/* hero button */}
+    <div className="mt-11 flex">
+      <Button 
+        text={"Learn More"}
+        variant={"btn-primary"}
+        onClick={handleLearnMore}
+      />
+      <button 
```

Comment:
### Tambahan (Opsional)
Buat kasih space antara icon sama button text, alternatif lain bisa pakai `gap` di className-nya button. Ini bisa dipake krn si `<button>` termasuk flex container. Daripada kasih margin ke tiap flex childnya, bisa coba pake `gap`. Praktis betul ini

**Contoh**
` <button onClick={handleWatchteaser} className="ml-4 font-body-1 btn-base flex gap-2"><img src={IPlay} />Watch The teaser</button>`